Packaging.props

```diff
@@ -1,5 +1,6 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <LicenseUrl>https://github.com/dotnet/corefx/blob/master/LICENSE</LicenseUrl>
```

Comment:
Oh fair I've been reviewing a lot of these so didn't really catch that. 

@richlander Should we point to the LICENSE file in the individual repo's?